grammars/level16-Additions.lark

```diff
@@ -3,7 +3,7 @@
 // TODO for tomorrow: this stills needs tests in level 16 for > and !=
 ?comparison_arg: var | text_in_quotes | NUMBER | list_access
 
-?atom: NUMBER | var | list_access | error_unsupported_number
+?atom: NUMBER | var_access | list_access | text_in_quotes
```

Comment:
I am not sure I understand why we are dropping `error_unsupported_number`?

Reply:
Ah I see, since we don't have unsupported numbers anymore! Not sure how this ended up in level 16? Bad case of the copy-pasties I guess?